recipes/simbody/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+source activate "${CONDA_DEFAULT_ENV}"
+
+mkdir build
+cd build
+cmake .. \
+	-LAH \
+	-DCMAKE_INSTALL_PREFIX="$PREFIX" \
+	-DBUILD_USING_OTHER_LAPACK="$PREFIX/lib/libopenblas.so" \
+	-DBUILD_VISUALIZER=off
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+	make -j$(nproc)
+	ctest -j$(nproc)
+	make -j$(nproc) install
```

Comment:
How should I set this up so that if I run the build on my local machine it uses `-j$(proc)`? Is there an environment variable that says whether I'm running on the conda-forge ci or not?